modules/programs/mpv.nix

```diff
@@ -122,7 +137,7 @@ in {
 
   config = mkIf cfg.enable (mkMerge [
     {
-      home.packages = [ pkgs.mpv ];
+      home.packages = [ (pkgs.mpv-with-scripts.override { scripts = cfg.scripts; }) ];
```

Comment:
Perhaps have something like
```
home.packages = [(
  if cfg.scripts == []
  then pkgs.mpv
  else pkgs.mpv-with-scripts.override { scripts = cfg.scripts; }
)];
```
to avoid the unnecessary wrapper when no scripts are used?